rts/lra/narayana-lra/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -142,6 +143,10 @@ public void filter(ContainerRequestContext containerRequestContext) {
             if (cancel0n.length != 0) {
                 containerRequestContext.setProperty(CANCEL_ON_PROP, cancel0n);
             }
+
+            if (transactional.timeLimit() != 0) {
```

Comment:
The original code  that you cut and pasted was clearer than your change:

``` java
        long timeLimit = transactional.timeLimit();
        ChronoUnit timeUnit = transactional.timeUnit();

        ... Duration.of(timeLimit, timeUnit).toMillis();
```

Also the original code was fine so why the updates?

Reply:
I can change this back, but recomputing timeout every time it's accessed is unnecessary.

Reply:
I meant that the having local variables for `timeLimit` and `timeUnit` improved readablity.

I realise that since we now need to pass the timeout into `getTerminationUris` it makes sense to cache the timeout value (so don't change it back to a method call). But it is fine if you want to keep the change and I will approve the PR as is.